olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -267,7 +267,7 @@ spec:
                       fieldPath: metadata.namespace
                 - name: METRICS_DOMAIN
                   value: knative.dev/serving-operator
-                image: registry.svc.ci.openshift.org/openshift/knative-v0.12.1:knative-serving-operator
+                image: quay.io/nak3/knative-v0.12.1:remove-servicemesh
```

Comment:
Note: We agreed to merge this as is and @nak3 will send a followup that will move this back to the "official" image to break the dependency cycle.